package.json

```diff
@@ -2,46 +2,45 @@
   "name": "@twilio-labs/serverless-toolkit",
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap --no-ci",
-    "postbootstrap": "lerna run build",
-    "release": "lerna version --conventional-commits --no-commit-hooks --no-push",
-    "npm:publish": "lerna publish from-git",
+    "bootstrap": "npm run build",
+    "build": "npm run build --workspaces --if-present",
+    "changeset": "changeset",
+    "version-packages": "npm run create-version && npm run version-package --workspaces --if-present",
+    "create-version": "changeset version",
+    "npm:publish": "changeset publish",
     "cm": "git-cz",
     "jest": "jest",
-    "build:noemit": "lerna run build:noemit",
-    "clean": "lerna run clean",
+    "build:noemit": "npm run build:noemit --workspaces --if-present",
+    "clean": "npm run clean --workspaces --if-present",
     "test": "run-s build:noemit jest",
```

Comment:
one thing I realized I should/could advise on here: I'm not sure what `run-s` is but this might be able to be replaced with npm commands if `build:noemit jest` are scripts in each workspace. Specifically `npm run build:noemit --workspaces && npm run jest --workspaces`.

Reply:
`run-s` is an alias from `npm-run-all` and runs those two comments in serial. It's a holdover from the time when `&&` wouldn't always work on Windows. It could probably go away now because I think `npm` scripts make `&&` work even on Windows. So the equivalent here would be `npm run build:noemit && npm run jest`